Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -839,6 +849,84 @@ final class VisibleItemsProvider {
     }
   }
 
+  private func correctedScrollToItemFrameForContentBoundaries(
```

Comment:
unclear to me what the correction we're applying here is

Reply:
https://github.com/airbnb/HorizonCalendar/pull/17/commits/8db3ad34e1e74dc15f841dfff6a541801736c40f added a detailed description for the purpose of this function - hoping that clears it up!